vpr/src/draw/search_bar.cpp

```diff
@@ -138,9 +138,11 @@ void search_and_highlight(GtkWidget* /*widget*/, ezgl::application* app) {
 bool highlight_rr_nodes(int hit_node) {
     t_draw_state* draw_state = get_draw_state_vars();
 
+    //TODO: fixed sized char array may cause overflow.
```

Comment:
@vaughnbetz I am not sure why there is a fixed-length message string here. But it looks dangerous to me. Just a reminder here. See if you do need to fix it.